### ISSUE TITLE
DocC drops content have single-line directive

### ISSUE THREAD

**Issue by franklinsch** [apple/swift-markdown#65]
### Description

When compiling this documentation comment:

```swift
/// Hello
///
/// Some content
///
/// @Comment { This is a comment }
///
/// Hello World
```

The text after the comment directive incorrectly gets dropped. This doesn't reproduce when writing the directive over multiple lines.

### Checklist

- [X] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/apple/swift-docc/issues).

### Expected Behavior

DocC should not drop content after single-line directives.

### Swift-DocC Version Information

Swift 5.7

**Comment by QuietMisdreavus** [comment]
I transferred @ktoso's original reporting of this problem to swift-markdown in #64.

**Comment by Kyle-Ye** [comment]
May be fixed by #66 
<img width="1154" alt="image" src="https://user-images.githubusercontent.com/43724855/183453904-85293a24-fe65-450e-96dc-a5320d52593f.png">

**Comment by QuietMisdreavus** [comment]
Closing this as a duplicate of #64.
